service/history/queues/executable.go

```diff
@@ -81,7 +81,8 @@ var (
 	// reschedulePolicy is the policy for determine reschedule backoff duration
 	// across multiple submissions to scheduler
 	reschedulePolicy                      = common.CreateTaskReschedulePolicy()
-	taskNotReadyReschedulePolicy          = common.CreateTaskNotReadyReschedulePolicy()
+	taskNotReadyReschedulePolicy          = common.CreateDeleteOpenExecReadyReschedulePolicy()
```

Comment:
any reason to use CreateDeleteOpenExecReadyReschedulePolicy instead of CreateTaskNotReadyReschedulePolicy

Reply:
+1